20.valid-parentheses.py

```diff
@@ -0,0 +1,27 @@
+#
+# @lc app=leetcode id=20 lang=python3
+#
+# [20] Valid Parentheses
+#
+
+# @lc code=start
+class Solution:
+    def isValid(self, s: str) -> bool:
+        """
+        time: O(len(s))
+        space: O(len(s))
+        """
+        stack = []
```

Comment:
use deque instead of list

```py
from collections import deque

class Solution:
    def isValid(self, s: str) -> bool:
        stack = deque()
```